package.json

```diff
@@ -41,6 +41,7 @@
     "@zxing/library": "^0.19.1",
     "bignumber.js": "^9.0.2",
     "bip32": "^4.0.0",
+    "bitcoin-address-validation": "^2.2.3",
```

Comment:
Maybe it is enough to use this regex and not to bring the external library?
`public addressValidationPattern: string = '^(?:[13]{1}[a-km-zA-HJ-NP-Z1-9]{25,34}|bc1[a-z0-9]{39,59})$'`
https://github.com/airgap-it/airgap-coin-lib/blob/912f326124ab9928bcc6b23f63cca55919a5b655/packages/bitcoin/src/v0/protocol/BitcoinProtocol.ts#L210

Reply:
you have right, regex is enough